src/app/pages/form/form.component.ts

```diff
@@ -1,9 +1,72 @@
 import { Component } from '@angular/core';
+import { FormBuilder, Validators } from '@angular/forms';
+
+interface City {
+    name: string;
+  //   code: string;
+}
 
 @Component({
-    selector: 'app-form',
-    templateUrl: './form.component.html'
+  selector: 'app-form',
+  templateUrl: './form.component.html',
+  styleUrls: ['./form.component.scss'],
 })
 export class FormComponent {
+  userForm = this.fb.group({
+    location: ['', Validators.required],
+    destination: [''],
+    selectedCity: [''],
+    departureDate: [''],
+    returnDate: [''],
+    passenger: [''],
+
+  });
+  citiesFrom: any[] | undefined;
+  citiesTo: any[] | undefined;
+    passenger: any[] | undefined;
+
+  constructor(private fb: FormBuilder) {}
+
+// departureDate: Date = new Date();
+//   returnDate: Date = new Date();
+
+
+  ngOnInit() {
+    this.citiesFrom = [
+      { name: 'New York, USA' },
+      { name: 'Rome, Italy' },
+      { name: 'London, UK' },
+      { name: 'Warsaw, Poland' },
+    ];
+
+    this.citiesTo = [
+      { name: 'Paris, France' },
+      { name: 'Alaska, USA' },
+      { name: 'Katowice, Poland' },
+      { name: 'Barcelona, Spain' },
+    ];
+
+    this.passenger = [
+        { name: '1 passenger'},
```

Comment:
tutaj zdecydowanie prościej i bardziej poprawnie by było jakby to była tablica typu `number`  i zawierałaby numery od 1 do 10 a nie stringi

Reply:
Powyżej już dodałam komentarz z dodaniem `passengers` :) usuwamy całego ngOnInita